spec/routing/hq/sponsors_routing_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Sponsors routing', type: :routing do
```

Comment:
I thought this way was better, if for nothing else because the method name is in the code, not in a comment.  It will appear in error messages.  But liking this way and previously receiving good feedback, I'll leave it unless one more tie-breaking vote comes in to change it to our aa-style routing specs.  @azangru @razorcd